README.md

```diff
@@ -60,6 +60,56 @@ locally and then register the TailwindCSS components.
 Alternatively, you can use the import lines below to import the
 individual features you need.
 
+### Alerts
+![Alerts](https://raw.githubusercontent.com/esmale/stimulus-components/master/alert.png)
```

Comment:
Could you swap this URL with `https://i.imgur.com/jxQ8k1t.png`? That way if you ever deleted your fork, it wouldn't break.

We probably need to upload all the images somewhere else so they won't accidentally disappear. 😬

Reply:
Updated!